src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,66 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String value = readFile(fromFileName);
+        String report = createReport(value);
+        fileWrite(toFileName, report);
+    }
+
+    public static String readFile(String fromFileName) {
+        File fl = new File(fromFileName);
+        try {
+            BufferedReader brd = new BufferedReader(new FileReader(fl));
+            StringBuilder strBuilder = new StringBuilder();
+            String value = brd.readLine();
```

Comment:
1. Please avoid using abbreviations, you will not remember meaning of them in a week.
2. Variable names should be based on what is contained inside of it, not type of variable
3. Please use try-with-resources for autoclosable items
```suggestion
        File inputFile = new File(fromFileName);
        try (BufferedReader reader = new BufferedReader(new FileReader(inputFile))) {
            StringBuilder fileContent = new StringBuilder();
            String currentLine = reader.readLine();
```